### PR TITLE
Add verbosity to errors in cpu.py

### PR DESCRIPTION
**Pull request by mgoin**

### CODE REVIEW

**Review by bfineran — APPROVED** [review]

**Review by tlrmchlsmth — APPROVED** [review]

File: src/deepsparse/cpu.py [hunk 1]
```diff
@@ -182,8 +184,10 @@ def cpu_architecture() -> architecture:
 
     if arch.isa not in VALID_VECTOR_EXTENSIONS:
         raise OSError(
-            "Neural Magic: The AVX instruction set is unknown. Set NM_ARCH to one of"
-            " {} to continue.".format(",".join(VALID_VECTOR_EXTENSIONS))
+            (
+                "Neural Magic: The AVX instruction set '{}' is unknown."
```
[the remark this answers is not in the document]
Comment (mgoin) [reply src/deepsparse/cpu.py]: Yeah good point I thought to revisit it once we actually have support but I scrubbed it out now. I also added a stipulation to the NM_ARCH recommendation

**Review by bnellnm — APPROVED** [review]

File: src/deepsparse/cpu.py [hunk 2]
```diff
@@ -154,36 +154,39 @@ def cpu_architecture() -> architecture:
         - detects the vector instruction set available (avx2, avx512)
         - if vnni is available
 
-    NM_ARCH environment variable can be used to override the avx instruction
+    NM_ARCH environment variable can be used to override the instruction
```
Comment (andy-neuma) [inline src/deepsparse/cpu.py:157]: cool

File: src/deepsparse/cpu.py [hunk 3]
```diff
@@ -77,7 +77,7 @@ def __init__(self, *args, **kwargs):
     def __setattr__(self, name: str, value: Any):
         if name != "__dict__":
             raise AttributeError(
-                "Neural Magic: Architecture: can't modify {}".format(name)
+                "Neural Magic: Architecture: can't modify {} to {}".format(name, value)
```
Comment (andy-neuma) [inline src/deepsparse/cpu.py:80]: should there be single quotes?

**Review by andy-neuma — APPROVED** [review]
thanks

**Pull request merged**